packages/vite/src/node/config.ts

```diff
@@ -205,6 +205,17 @@ export interface UserConfig {
       'plugins' | 'input' | 'onwarn' | 'preserveEntrySignatures'
     >
   }
+  /**
+   * Whether the app is an SPA.
+   *
+   * This toggles the SPA catch-all routing of the development and preview
+   * server: by default, all URLs are rendered to the same single page (in order
+   * to enable SPA client-side routing). For SSR and MPA apps, this behavior can
+   * be disabled by setting `isSPA` to `false`.
+   *
+   * @default true
```

Comment:
the other options don't have a new line before `@default`
```suggestion
   * Whether the app is a SPA.
   * This toggles the SPA catch-all routing of the development and preview
   * server: by default, all URLs are rendered to the same single page (in order
   * to enable SPA client-side routing). For SSR and MPA apps, this behavior can
   * be disabled by setting `isSPA` to `false`.
   * @default true
```